BaseFilter/sections/antiadblock.txt

```diff
@@ -1007,6 +1007,10 @@ pornhub.com,pornhub.org,pornhub.net#%#//scriptlet('prevent-setTimeout', '\x41\x6
 !
 ! NOTE: Regular rules
 !
+! https://github.com/AdguardTeam/AdguardFilters/issues/169805

```

Comment:
I'm sorry, didn't mention. It's should be in the foreign filter

Reply:
Don't worry, my fault 😊
Fixed